Quantum-Control-Applications/Superconducting/Single Flux Tunable Transmon/SSB mixer for readout/calibrations.py

```diff
@@ -0,0 +1,80 @@
+"""
+calibrations.py: template for easily performing single qubit calibration protocols.
+"""
+
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from configuration import *
+from qualang_tools.addons.calibration.calibrations import QUA_calibrations
+
+
+# Relevant configuration parameters
+#############################################################
+resonator_element = "resonator"
+resonator_operation = "readout"
+qubit_element = "qubit"
+qubit_operation = "gauss"
+flux_line_element = "flux_line"
+flux_line_operation = "const"
+int_w = ["cos", "sin", "minus_sin", "cos"]
+outs = ["out1", "out2"]
+#############################################################
+
+# Initialize the calibration API with the relevant configuration parameters
+my_calib = QUA_calibrations(
+    configuration=config,
+    readout=(resonator_element, resonator_operation),
+    qubit=(qubit_element, qubit_operation),
+    integration_weights=int_w,
+    outputs=outs,
+)
+
+n_avg = 1000
+# Resonator spectroscopy scan
+scans = [
+    ("frequency", np.arange(100e6, 300e6, step=0.1e6)),
+]
+my_calib.set_resonator_spectroscopy(scan_variables=scans, iterations=n_avg, cooldown_time=0)
```

Comment:
The idea is to always comment in the calibration that you want?
Maybe let's put it all in an `if` `elif` `else` clause?

Reply:
Yes, I did not find a good idea to select the calibrations... Since you can run several calibrations in a row I was not found of the `if` `else`, I thought about putting the calibrations you want to run in a list and iterate over it, but I think it complicates things for very little gain... WDYT?

Reply:
Hmm, I see.
How about this:
1. Rename the "scans" to be specific: "resonator_spectrscopy_scan"
2. Move all of the scan parameters to be together.
3. Move all of the calibs to be togeher, and add a comment in the beginning: `# Please uncomment any calibrations you wish to run`
What do you think?